src/type-system.ts

```diff
@@ -232,33 +232,30 @@ type ElysiaFile = (
 	options?: Partial<ElysiaTypeOptions.Files> | undefined
 ) => TUnsafe<File>
 
-const File: ElysiaFile =
-	(TypeRegistry.Get('Files') as unknown as ElysiaFile) ??
```

Comment:
Here was "Files" instead of "File"

Reply:
Furthermore, `TypeRegistry.Get('...')` didn't work as expected because it returns the validation function instead of the schema, which means it breaks if someone overwrites it nowadays.